packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -121,6 +125,20 @@ export const components = ({
             outlineStyle: "solid",
             zIndex: 1,
           },
+          svg: {
+            fontSize: "1.2em",
+            height: "1em",
+            position: "relative",
+            insetBlockStart: "-0.0625em",
+            verticalAlign: "middle",
+            width: "1em",
```

Comment:
What's this margin for `insetBlockStart`? Can we use design tokens for these values? Any sort of spacing should be in there to match everything else in the design system or we might wanna ask Alex why this is a one-off.

Reply:
this is an existing logic that overrode styles for svgs inside MuiLink but was overriding all svgs. I just moved and made it more specific to only apply to svg.